content/tutorials/mushr_navigation_system.md

```diff
@@ -0,0 +1,167 @@
+$ rosrun nhttc_ros route_publisher.py
+{{< / highlight >}}
+
+You should see something like this on rviz:
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/demo_default.gif" width="800" >}} <br>                           
+<br>
+This should be considered the default case henceforth.
+
+# API
+
+The nhttc_ros wrapper has the following parameters:
+#### car_name (string):
+The name assigned to the car (can be any string but prefer car{index number})
+
+#### solver_time (int):
+The maximum time in milliseconds for which the solver is allowed to run
+
+#### max_ttc (float):
+The maximum time-to-collision used by the solver: any agents that have a time to collision larger than this will not be considered in the cost function
+
+#### carrot_goal_ratio (float):
+The ratio of the carrot-goal/lookahead distance to the turning radius. value of 1 means that the lookahead distance is the same as the turning radius.
+
+#### obey_time" value(boolean):
+Set to true if the car is supposed to adhere to the time coordinate of the waypoints (as in, to arrive at a waypoint at a given time and not before/after).
+
+#### allow_reverse (boolean):
+Set to true if the car is allowed to go in reverse.
+
+### Publishers
+Topic | Type | Description
+------|------|------------
+`/car_name/mux/ackermann_cmd_mux/input/navigation` | [ackermann_msgs/AckermannDriveStamped](http://docs.ros.org/en/jade/api/ackermann_msgs/html/msg/AckermannDriveStamped.html)| steering and speed control of car corresponding to car_name.
+`/car_name/cur_goal` | [geometry_msgs/PoseStamp](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseStamped.html)| topic on which current waypoint is published.
+`/car_name/time_goal` | [geometry_msgs/PoseStamp](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseStamped.html)| waypoint being used for timing purposes.
+
+### Subscribers
+Topic | Type | Description
+------|------|------------
+`/car1/car_pose` | [geometry_msgs/PoseStamp](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseStamped.html)| position of car 1
+...
+`/car{n}/car_pose` | [geometry_msgs/PoseStamp](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseStamped.html)| position of car {n}
+`/car_name/waypoints` | [geometry_msgs/PoseArray](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseArray.htmll)| waypoint array corresponding to car_name. The z axis coordinate represents the time difference between 2 waypoints.
+
+Note that the z axis data in `/car/waypoints` topic represents the time difference between two consecutive waypoints in a unitless fashion, and the number should be pre-multiplied by 0.001 before publishing (so that the visualization on rviz does not look elevated). 1 unit of time here is equal to (distance between two waypoints/expected speed of the agent). If the car moves at 0.5 m/s and the distance between two waypoints is 1 meter, then 1 unit of time would equal to 2 seconds. 
+
+## Tuning the parameters
```

Comment:
update text to tell people to look at the config file not the launch file. Also the highlighted params should change from xml format to yaml